docs/XBlocks.md

```diff
@@ -0,0 +1,217 @@
+# Group Project building blocks
+
+The following blocks are a part of Group Project XBlock v2.
+
+## Top-level XBlocks
+
+* ` Group Project` - top level XBlock, represents entire group project as a whole
```

Comment:
@e-kolpakov No need to keep both "entire" and "as a whole" (I'd remove "entire").